src/Workspaces/Core/Portable/Workspace/Solution/Checksum.cs

```diff
@@ -121,17 +73,29 @@ public override bool Equals(object obj)
         public override int GetHashCode()
             => _checksum.GetHashCode();
 
-        public override unsafe string ToString()
+        public unsafe string ToBase64String()
```

Comment:
💡 The `unsafe` is only needed in one of the target frameworks. Consider removing it from the signature and adding an `unsafe{}` block in the implementation so it's clear we are moving away from the requirement.

Reply:
sure